### PR TITLE
Simplify installer

### PR DESCRIPTION
**Pull request by fractaledmind**
This matches the structure that Solid Queue now uses (after https://github.com/rails/solid_queue/pull/321) and Solid Cache now uses (after https://github.com/rails/solid_cache/pull/202)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,19 +20,57 @@ Or install it yourself as:
...
 ```bash
 $ bin/rails generate solid_cable:install
 ```
 
-If you want to install to a different database you can pass an env variable.
-```bash
-$ DATABASE=cable bin/rails generate solid_cable:install
+This will create the `db/cable_schema.rb` file.
+
+You will then have to add the configuration for the errors database in `config/database.yml`. If you're using sqlite, it'll look something like this:
```
Comment (npezza93) [inline README.md:31]:
```suggestion
You will then have to add the configuration for the database in `config/database.yml`. If you're using SQLite, it'll look something like this:
```

File: README.md [hunk 2]
```diff
@@ -20,19 +20,57 @@ Or install it yourself as:
...
+  cable:
+    <<: *primary_production
+    database: app_production_cable
+    migrations_paths: db/cable_migrate
 ```
 
-Update `config/cable.yml` to use the new adapter. connects_to is can be omitted
-if you want to use the primary database.
+> [!NOTE]
+> Calling `bin/rails generate solid_cable:install` will automatically setup `config/cable.yml`, so no additional configuration is needed there (although you must make sure that you use the `cable` name in `database.yml` for this to match!). But if you want to use Solid Cable in a different environment (like staging or even development), you'll have to manually add that `connects_to` block to the respective environment in the `config/cable.yml`file. And, as always, make sure that the name you're using for the database in `config/cable.yml` matches the name you define in `config/database.yml`.
```
Comment (npezza93) [inline README.md:60]:
```suggestion
> Calling `bin/rails generate solid_cable:install` will automatically setup `config/cable.yml`, so no additional configuration is needed there (although you must make sure that you use the `cable` name in `database.yml` for this to match!). But if you want to use Solid Cable in a different environment (like staging or even development), you'll have to manually add that `connects_to` block to the respective environment in the `config/cable.yml` file. And, as always, make sure that the name you're using for the database in `config/cable.yml` matches the name you define in `config/database.yml`.
```

**Review by npezza93 — CHANGES_REQUESTED** [review]
Few text tweaks then this is good to go! Thanks!

**Comment by fractaledmind** [comment]
@npezza93: Thanks. Updated

**Review by npezza93 — APPROVED** [review]

**Pull request merged**